pkg/distro/rhel/azure.go

```diff
@@ -0,0 +1,52 @@
+package rhel
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+
+	"github.com/osbuild/images/internal/common"
+	"github.com/osbuild/images/pkg/customizations/fsnode"
+	"github.com/osbuild/images/pkg/osbuild"
+)
+
+// Dataloss warning script for Azure images.
+//
+//go:embed temp-disk-dataloss-warning.sh
+var azureDatalossWarningScriptContent string
+
+// Returns a filenode that embeds a script and a systemd unit to run it on
+// every boot.
+// The script writes a file named DATALOSS_WARNING_README.txt to the root of an
+// Azure ephemeral resource disk, if one is mounted, as a warning against using
+// the disk for data storage.
+// https://docs.microsoft.com/en-us/azure/virtual-machines/linux/managed-disks-overview#temporary-disk
+func CreateAzureDatalossWarningScriptAndUnit() (*fsnode.File, *osbuild.SystemdUnitCreateStageOptions, error) {
+	datalossWarningScriptPath := "/usr/local/sbin/temp-disk-dataloss-warning"
+	datalossWarningScript, err := fsnode.NewFile(datalossWarningScriptPath, common.ToPtr(os.FileMode(0755)), nil, nil, []byte(azureDatalossWarningScriptContent))
+	if err != nil {
+		return nil, nil, fmt.Errorf("rhel9/azure: error creating file node for dataloss warning script: %w", err)
```

Comment:
The `rhel9/azure:` is no longer accurate, is it?

Reply:
Fixed